man/man8/zfs.8

```diff
@@ -58,12 +58,31 @@ The
 command configures ZFS datasets within a ZFS storage pool, as described in
 .Xr zpool 8 .
 A dataset is identified by a unique path within the ZFS namespace.
+.Pp
+.Dl [pool]/[component/]*[component]
```

Comment:
```
... the ZFS namespace:
.D1 Ar pool Ns Oo Ar component Oc Ns …
for example
.Dl rpool/var/log
```

Reply:
The proposed change above produces underscored
```
pool[component]
```
which looks even more confusing.

Reply:
oh, i forgot the slash, it was late, i meant
```
.D1 Ar pool Ns Oo Sy / Ns Ar component Oc Ns …
```
to render as CI(pool)R([)B(/)CI(compoment)R(]…)

Reply:
Ok, I got the point now. To be honest it is my first man page modification ever.